### DISCUSSION TITLE
Swagger question

### DISCUSSION THREAD

**Discussion by stuartj1-1981** [bilan/connectlife-api-connector#39] [General]
Hi

I'm trying to use the swagger link you posted to connect to my Hisense tumble dryer.  I keep failing with wrong AUID format.  Do you have any tips on how to find this?

PS thanks for your work on the integration.  I have installed and can see the device talking to MQTT but cannot create any entities so trying to reverse engineer.

**Comment by bilan** [comment]
Hi,
unfortunately I don't have any idea about solving your AUID problem but do you have the latest Homeasistant and the add-on in v2.1.11?

**Reply by stuartj1-1981** [reply]
Yes v2.1.11 is installed.  I am contacting Hisense for the aud number.

**Comment by Monacoslo** [comment]
Post a photo of your product label

**Reply by stuartj1-1981** [reply]
Morning

Label on drum and label on back.

![IMG_1689](https://github.com/bilan/connectlife-api-connector/assets/103891503/94bc2bcb-1f52-4ac4-a7c2-2887e4767e5d)
![IMG_1691](https://github.com/bilan/connectlife-api-connector/assets/103891503/91f7dde9-b2db-486e-8518-5d01ef3c4c0f)

Thanks

Stu

**Comment by Monacoslo** [comment]
And what are you trying to do with swagger?

**Reply by stuartj1-1981** [reply]
trying to connect to the dryer to see what endpoints I can see and then connect to them via MQTT or REST.  The current integrations are not currently working for me so I am trying to see if I can fix it.

**Comment by Monacoslo** [comment]
You can get all the data from log of the addon. Swagger does not provide this data anymore since it was blocked

**Reply by stuartj1-1981** [reply]
Arrr ok didn't realise it had been blocked.  I will revert back to the bilans add-on log then.  Thanks

**Reply by stuartj1-1981** [reply]
I think I have found why bilans integration doesn't work for me .  The type code for the dryer is 30

INFO: Skipping device with unknown type code: xxxxxxxxxxxxxxxx {"wifiId":"xxxxxxxxxxx","deviceId":"xxxxxxxxxxxxx","puid":"xxxxxxxxxxxxxx","deviceNickName":"Tumble dryer"
